dev/com.ibm.ws.session.cache/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -14,3 +14,12 @@
 #NLS_ENCODING=UNICODE
 #NLS_MESSAGEFORMAT_NONE
 #
+
+libraryRef=Shared Library
+libraryRef.desc=Identifies JCache provider files.
+
+uri=JCache Configuration URI
+uri.desc=Vendor specific JCache configuration URI, which is passed to the JCache provider when obtaining the cacheManager.
```

Comment:
cacheManager should not be camel case.  Probably either CacheManager or cache manager.

Reply:
I'll switch it to CacheManager.